app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotVaccinatedError, \
```

Comment:
You should use `()` instead of `\`:
```python
from app.errors import (NotVaccinatedError,
                        OutdatedVaccineError,
                        NotWearingMaskError)
```
or
```python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)

Reply:
Thanks, for the good advice!